test/test_image.py

```diff
@@ -0,0 +1,473 @@
+import os
+from shlex import split
+from subprocess import PIPE, CalledProcessError, run
+
+import pytest
+
+from docker_cli.exceptions import CommandNotFoundOnImageError
+from docker_cli.image import Image, get_image_id
+
+
+@pytest.fixture
+def test_image_id():
```

Comment:
Fixture names probably shouldn't be prefixed with `test_` because that's the naming convention that pytest uses for test discovery, so I think pytest will try to run this function as a test, which wasn't what you intended.

```suggestion
def image_id():
```